### PR TITLE
Set up array to display places in the Itinerary

### PR DESCRIPTION
**Pull request by Zarthia**

### CODE REVIEW

File: src/itinerary/index.tsx [hunk 1]
```diff
@@ -27,49 +26,42 @@ const Itinerary = ({
...
-        <span>{arrival.mainLabel}</span>
-        {
-          (!small && arrival.subLabel) && (
-            <span className="kirk-itinerary-subtext">{arrival.subLabel}</span>
-          )
-        }
-      </div>
-    </li>
+    {
+      places.map((place, index) => (
```
Comment (pith) [inline src/itinerary/index.tsx]: So you display the first and the last twice ?

Why not:

const [first, rest, last] = places

Comment (Tuxosaurus) [reply src/itinerary/index.tsx]: I don't see how it could be displayed twice?

File: src/itinerary/story.tsx [hunk 2]
```diff
@@ -20,20 +20,22 @@ stories.add(
...
-          time: text('Time from', '09:00'),
-        }}
-        arrival={{
-          mainLabel: text('Main label to', 'Bordeaux'),
-          subLabel: text('Sublabel to', 'Gare Bordeaux Saint-Jean'),
-          isoDate: '2017-12-11T12:00',
-          distanceFromPoint: isToDistance && '8km',
-          time: text('Time to', '12:00'),
-        }}
+        places={[
```
Comment (loick) [inline src/itinerary/story.tsx]: Maybe you should add a test including the stopovers

File: src/itinerary/index.tsx (hunk 1 above)
Comment (loick) [reply src/itinerary/index.tsx]: @pith This wouldn't work :)
`const [first, rest, last] = places`

**Pull request merged**